osquery/main/harnesses/gen_fuzz_config_dict.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# If this is not commented out, we get 'sed' commands in our output below
+#set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+function usage() {
+    echo "${BASH_SOURCE[0]} destination-file"
+}
+
+function main() {
+    if [[ $# < 1 ]]; then
+	usage
+	exit 1
+    fi
+
+    # dict format is keyword="value" or just "value" - so we need to make sure our output is quoted.
+    
+    egrep -h -R -o "HasMember\\(\"([^\"]+)\"\\)" *  | sed 's/HasMember(//' | sed 's/)//' > tmp
```

Comment:
```suggestion
    egrep -h -R -o "HasMember\\(\"([^\"]+)\"\\)" ./  | sed 's/HasMember(//' | sed 's/)//' > tmp
```

As this is `-R` I think `./` is more clear than `*`

Reply:
Sure